### ISSUE TITLE
I can't use the neon theme (Command 'Synthwave '84: Disable Neon Drems' resulted in an error)

### ISSUE THREAD

**Issue by Joanseco** [robb0wen/synthwave-vscode#326]
[
![Captura de pantalla 2024-10-09 045526](https://github.com/user-attachments/assets/f2c52e2e-325f-4fc5-86e7-16ec801a0d6c)
](url)
![Captura de pantalla 2024-10-09 045658](https://github.com/user-attachments/assets/ec9bb449-a816-436f-9a5a-e96fd1f5615c)

**Comment by jqweryjson** [comment]
Same issue. 
`2024-10-09 13:37:44.839 [error] TypeError: Cannot read properties of undefined (reading 'filename')
	at /home/kiril/.vscode/extensions/robbowen.synthwave-vscode-0.1.15/src/extension.js:27:44
	at T0.h (file:///usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:114:32845)
	at T0.$executeContributedCommand (file:///usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:114:33695)
	at Zb.S (file:///usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:31:113896)
	at Zb.Q (file:///usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:31:113676)
	at Zb.M (file:///usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:31:112765)
	at Zb.L (file:///usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:31:111870)
	at gh.value (file:///usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:31:110667)
	at T.B (file:///usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:29:732)
	at T.fire (file:///usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:29:950)
	at no.fire (file:///usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:31:9399)
	at gh.value (file:///usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:174:13273)
	at T.B (file:///usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:29:732)
	at T.fire (file:///usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:29:950)
	at no.fire (file:///usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:31:9399)
	at MessagePortMain.<anonymous> (file:///usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:174:11562)
	at MessagePortMain.emit (node:events:519:28)
	at MessagePortMain.emit (node:domain:488:12)
	at MessagePortMain._internalPort.emit (node:electron/js2c/utility_init:2:2619) synthwave84.enableNeon {"value":"RobbOwen.synthwave-vscode","_lower":"robbowen.synthwave-vscode"}`

**Comment by Rottbott** [comment]
I was able to work around this and enable the glow with these changes to extension.js:
```diff
-		const appDir = path.dirname(require.main.filename);
-		const base = appDir + (isWin ? "\\vs\\code" : "/vs/code");
+		const appDir = path.dirname(vscode.env.appRoot);
+		const base = appDir + (isWin ? "\\app\\out\\vs\\code" : "/app/out/vs/code");
```
and
```diff
		const htmlFile =
			base +
			(isWin
-				? "\\"+electronBase+"\\workbench\\workbench.html"
-				: "/"+electronBase+"/workbench/workbench.html");
+				? "\\"+electronBase+"\\workbench\\workbench.esm.html"
+				: "/"+electronBase+"/workbench/workbench.esm.html");
```
Don't forget to change it in `uninstall()` as well. I have no idea whether this is the right thing to do, but it works for me.

**Comment by Bishop1988** [comment]
@Rottbott This worked for me too. Cheers!

**Comment by easylee1996** [comment]
@Rottbott It works for me. Thanks!

**Comment by vinayakkulkarni** [comment]
temporary fix for v1.94 https://github.com/robb0wen/synthwave-vscode/issues/317#issuecomment-2351804426

fix for v1.95 - https://github.com/robb0wen/synthwave-vscode/pull/318/files

**Comment by UR-Hiro** [comment]
can you teach me how to fix the code. My brain cannot function without the glow.

**Comment by robb0wen** [comment]
This issue should now be fixed in v0.1.16
